src/file.ts

```diff
@@ -1698,7 +1701,7 @@ class File extends ServiceObject<File> {
       pumpify([
         gzip ? zlib.createGzip() : through(),
         validateStream,
-        fileWriteStream,
+        fileWriteStream.on('progress', evt => stream.emit('progress', evt)),
```

Comment:
Can you put this event handler assignment on its own line? Just makes it easier to see what the code does line-by-line.